packages/connect-voting-disputable/src/models/Vote.ts

```diff
@@ -98,4 +153,25 @@ export default class Vote {
   onCollateralRequirement(callback: Function): SubscriptionHandler {
     return this.#connector.onCollateralRequirement(this.id, callback)
   }
+
+  async setting(): Promise<Setting> {
+    return this.#connector.setting(this.settingId)
+  }
+
+  onSetting(callback: Function): SubscriptionHandler {
+    return this.#connector.onSetting(this.settingId, callback)
+  }
+
+  _disputableVoting(): DisputableVoting {
+    return new DisputableVoting(this.#connector, this.votingId)
+  }
+
+  async _tokenDecimals(): Promise<string> {
+    const erc20 = await this._disputableVoting().token()
+    return erc20.decimals
+  }
+
+  _isValuePct(value: BigNumber, total: BigNumber, pct: BigNumber): boolean {
```

Comment:
What you think if change the naming of this function? I got a bit confuse in relation to what it does.